tests/microos/workloads/kea-container/dhcp4_client.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE"s openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: podman kea-container
+# Summary: install and verify Kea DHCP server container.
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use lockapi;
+use Utils::Systemd qw(disable_and_stop_service systemctl check_unit_file);
+
+sub run {
+    my ($self) = @_;
+    mutex_wait 'barrier_setup_done';
+
+    assert_script_run("ip a");
```

Comment:
Is this necessary or does this provide debugging information? If the later, consider using `record_info` instead, e.g.

    record_info("ip a", script_output("ip a"));